confd/warp/configReader.go

```diff
@@ -0,0 +1,107 @@
+package warp
+
+import (
+	"context"
+	"log"
+
+	"sort"
+
+	. "github.com/cloudogu/ces-confd/confd"
```

Comment:
**[checkstyle]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
should not use dot imports